src/processor.ts

```diff
@@ -35,27 +37,51 @@ import {
 import { handleBalanceTransfer } from './mappings/balances'
 import { handleContractEvent } from './mappings/nabla'
 import { handleUpdatedPrices } from './mappings/prices'
+import {
+    saveBlock,
+    saveCall,
+    saveEvent,
+    saveExtrinsic,
+} from './mappings/block-details'
 
 const DataSelection = { data: { event: true } } as const
 
 const processor = new SubstrateBatchProcessor()
     .setDataSource(config.dataSource)
     .setFields({
-        event: {},
+        block: {
+            timestamp: true,
+            digest: true,
+            extrinsicsRoot: true,
+            stateRoot: true,
+            validator: true,
+        },
         call: {
+            name: true,
+            args: true,
             origin: true,
             success: true,
             error: true,
         },
+        event: {
+            name: true,
+            args: true,
+            phase: true,
+        },
         extrinsic: {
             hash: true,
+            success: true,
+            error: true,
             fee: true,
+            signature: true,
             tip: true,
-        },
-        block: {
-            timestamp: true,
+            version: true,
         },
     })
+    .addCall({
+        extrinsic: true,
+        stack: true,
```

Comment:
Just curious, any idea what `stack` in the selector is?

Reply:
In the [documentation](https://docs.subsquid.io/substrate-indexing/setup/data-requests/#calls) it says that 

> with stack = true it will add all calls in the stack of each matching call, including itself, to the calls iterable;

but I honestly don't really understand what that is supposed to mean as it's too ambiguous.

Reply:
The 'base template' is [using it like this](https://github.com/subsquid-labs/giant-squid-explorer/blob/a3d9b4f08ecbd918f18c87fc715e742cbdf145c4/src/processor.ts#L60-L62) so it should be safe if we do too. But I can't tell the actual implications